src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleShell.cs

```diff
@@ -40,7 +40,7 @@ public static int Start(RunspaceConfiguration configuration, string bannerText,
 
         /// <summary>
         /// 
-        /// Entry point in to ConsoleShell. This method is called by the MSHManagedEntrace
+        /// Entry point in to ConsoleShell. This method is called by the MSHManagedEntrace (SP!)
```

Comment:
I think this is called from multiple places now:

https://github.com/PowerShell/PowerShell/blob/master/src/powershell-native/nativemsh/pwrshcommon/pwrshcommon.cpp#L1462

and

https://github.com/PowerShell/PowerShell/blob/master/src/powershell/Program.cs#L22